src/styles/index.scss

```diff
@@ -1,3 +1,100 @@
+@import 'fonts';
+@import 'stars';
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: $fontRoboto;
+  font-size: 12px;
+  box-sizing: border-box;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 16px;
+  padding-top: 32px;
+  background-color: $basicWhite;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    width: 160px;
+    max-width: 100%;
+    height: 135px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin: 0;
+    padding: 0;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 5px;
+    color: $mainAcsent;
+  }
+
+  &__code {
+    font-size: 10px;
+    align-self: start;
+    margin-bottom: 17px;
+    color: $secondaryColor;
+  }
```

Comment:
You've set `align-self: start;` on line 46. However, the parent element has `align-items: center;` (line 25). This means that this item will not be centered, unlike other items in the flex container. If it's intentional and you want this item to be aligned to the start, then it's fine. But if you want all items to be centered, you should remove `align-self: start;`.